source/application/DbgHelp/DbgHelpStackWalkCallback.cpp

```diff
@@ -13,15 +13,16 @@
 #include <atlbase.h>
 #include <atlcom.h>
 
-#include "application/logging.h"
 #include "DbgHelpPrivate.h"
 #include "DbgHelpStackWalkCallback.h"
 #include "DiaEnumIterator.h"
+#include "application/logging.h"
 
 namespace
 {
     /*
-     * In these functions length MUST match the "expected" size of the basic types (see DbgHelpBasicType::GetSize()),
+     * In these functions length MUST match the "expected" size of the basic types (see
+     * DbgHelpBasicType::GetSize()),

```

Comment:
This was an unfortunate break, you might need to fix this one manually.

Reply:
Disabled comment reflow.